src/NServiceBus.InMemoryPersistence.AcceptanceTests/ConfigureAcceptanceTestTransport.cs

```diff
@@ -0,0 +1,62 @@
+﻿
```

Comment:
Should this be moved to NServiceBus.AcceptanceTesting or NServiceBus.AcceptanceTests instead? Otherwise every downstream that wants to use this transport will have to write their own copy of this class. For our existing tests, downstreams only have to provide one of these for the component that's actually being tested in the repo.

That does add some complexity to how to set up the native pubsub delayed delivery values, but I think we should at least see if we can come up with a solution.

Reply:
yes, makes sense to move this to `NServiceBus.AcceptanceTests `. We cannot move this to the framework itself, as this part of the acceptance tests configuration is specific to our acceptance tests and is all part of the sources package.